airbyte-config/models/src/main/resources/types/ReplicationAttemptSummary.yaml

```diff
@@ -9,17 +9,29 @@ required:
   - bytesSynced
   - startTime
   - endTime
+  - totalStats
+  - streamStats
 additionalProperties: false
 properties:
   status:
     "$ref": ReplicationStatus.yaml
-  recordsSynced:
+  recordsSynced: # TODO (parker) remove in favor of totalRecordsEmitted
     type: integer
     minValue: 0
-  bytesSynced:
+  bytesSynced: # TODO (parker) remove in favor of totalBytesEmitted
     type: integer
     minValue: 0
   startTime:
     type: integer
   endTime:
     type: integer
+  totalStats:
+    "$ref": SyncStats.yaml
+  streamStats:
+    "$ref": "#/definitions/streamNameToSyncStats"
+    description: map of stream name to sync stats
+definitions:
+  streamNameToSyncStats:
```

Comment:
based on our conversation today i think the list makes sense. curious to see if you find anything to support the other side.